src/game/AI/ScriptDevAI/scripts/eastern_kingdoms/naxxramas/boss_four_horsemen.cpp

```diff
@@ -142,6 +148,39 @@ struct boss_horsmenAI : public ScriptedAI
             m_instance->SetData(TYPE_FOUR_HORSEMEN, SPECIAL);
     }
 
+    void EnterEvadeMode() override
+    {
+        ScriptedAI::EnterEvadeMode();
+        if (Creature* blaumeux = m_instance->GetSingleCreatureFromStorage(NPC_BLAUMEUX))
+        {
```

Comment:
These brackets can be removed. Also if you do smth like
std::vector<uint32> entries = {NPC_BLAUMEUX};
for (uint32 entry : entries)
getsingle
isincombat
enterevade

you end up with a lot less code.